playground.js

```diff
@@ -0,0 +1,28 @@
+const form = document.getElementById("form");
+
+form.addEventListener("submit", function (e) {
+  e.preventDefault();
+  if (form === null) {
+    console.error("form element not found");
+    return;
+  }
+
+  const fd = new FormData(form);
+
+  const [replacement, lowercase, trim, fallback] = Array.from(fd.values());
```

Comment:
Ah, this is why I thought there was a bug. `fd.values()` does not guarantee an order. Or if it does, it's not the order expected here. If I uncheck the `Lowercase` checkbox, `fallback` ends up being set to `false` rather than `lower`.

Reply:
ah I didn't realise an 'undefined' value results in the values being out of order. Let me refactor this.

Reply:
Sorry about this silly mistake. This is what happens when one is used to libraries like Yup and is programming in vanilla JS for once!